### PR TITLE
{tools}[GCCcore/12.3.0] PBZIP2 v1.1.13

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
@boegelbot: please test @ generoso

**Comment by jfgrimm** [comment]
@boegelbot: please test @ jsc-zen3

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
gpu07.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, 2 x NVIDIA NVIDIA A40, 535.86.10, Python 3.6.8
See https://gist.github.com/jfgrimm/3f25fac64c2148ae384a9fdc475687a2 for a full test report.

File: easybuild/easyconfigs/p/PBZIP2/PBZIP2-1.1.13-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+skipsteps = ['configure']
+
+installopts = 'PREFIX=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ['bin/pbzip2', 'share/man/man1/pbzip2.1'],
+    'dirs': [],
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/p/PBZIP2/PBZIP2-1.1.13-GCCcore-12.3.0.eb:39]: Can we add a sanity check command? Maybe something like this:
```suggestion

sanity_check_commands = ['pbzip2 -h 2>&1 | grep "Parallel BZIP2 v%(version)s"']

```

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/e1454f3a6ec19a6e945c3008e8d55375 for a full test report.

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @jfgrimm!

**Comment by emdrago** [comment]
Test report by @emdrago
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
gput052 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz, 1 x NVIDIA NVIDIA GeForce RTX 2080 Ti, 535.54.03, Python 3.9.13
See https://gist.github.com/emdrago/3729452d9411286fe8d15bd627eae272 for a full test report.

**Pull request merged**